apps/communications/contacts/test/unit/mock_vcard_reader.js

```diff
@@ -1,7 +1,11 @@
 'use strict';
 
 function MockVCardReader(textToParse) {
```

Comment:
nit: this parameter should no longer be named 'textToparse', it should be called 'data' or something similar and you should add a comment indicating what is intended to